pkg/git/git.go

```diff
@@ -68,6 +69,18 @@ func (g *Git) Push(workingDir string, args ...string) error {
 	return runCommand(workingDir, command)
 }
 
+// GetPushURL returns the push url with a token inserted
+func (g *Git) GetPushURL(remote string, token string) (string, error) {
+	pushURL, err := exec.Command("git", "remote", "get-url", "--push", remote).Output()
+	if err != nil {
+		return "", err
+	}
+
+	pushURLArray := strings.SplitAfter(strings.TrimSpace(string(pushURL)), "https://")
```

Comment:
This probably won't work for SSH URLs. Please add a test that verifies the functionality for both, SSH and HTTPS URLs.

Reply:
You're right regarding SSH. That's not supported and a token is needed for pushing. chart-releaser is usually used with GitHub Actions which don't use SSH anyways.